ios/Elements/RNSVGGroup.m

```diff
@@ -46,6 +48,9 @@ - (void)renderGroupTo:(CGContextRef)context rect:(CGRect)rect
             }
 
             [svgNode renderTo:context rect:rect];
+            
+            CGRect nodeRect = svgNode.clientRect;
+            [rects addObject:[NSValue valueWithCGRect:nodeRect]];
```

Comment:
Seems the unions could be done once here, instead of looping twice. (Same as android)